### PR TITLE
TRUNK-5259: Changed install.header.caption from OpenMRS {0} Installat…

### PR DESCRIPTION
**Pull request by klambo94**
…ion Wizard -> OpenMRS Core {0} Installation Wizard

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -2905,7 +2905,7 @@ install.complete.test.details=Other Details
 install.complete.test.settings=Currently installed database server settings
 install.complete.settings=Test installation settings
 
-install.header.caption=OpenMRS {0} Installation Wizard
+install.header.caption=OpenMRS Core {0} Installation Wizard
```
Comment (dkayiwa) [inline api/src/main/resources/messages.properties]: Can you replace Core with Platform?

**Comment by klambo94** [comment]
Implemented Changes - Waiting for verification before merging to master.

**Comment by dkayiwa** [comment]
Can u squash the commits as advised here https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by klambo94** [comment]
I've tried to squash the commits into one but it seems that I have just added 2 new commits while attempting to do this. As it is late, I am going to head off to get some rest and take a look at this on Saturday

**Comment by klambo94** [comment]
As my attempts to squash commits were leading to even more commits I closed this pull request and deleted the branch so I can create a new branch with the same name for the change. The new pull request can be found here: [TRUNK-5259](https://github.com/openmrs/openmrs-core/pull/2617)

**Pull request closed without merging**